server/service/src/document/raw_document.rs

```diff
@@ -22,6 +22,7 @@ pub struct RawDocument {
     pub owner_name_id: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub context: Option<String>,
+    pub is_sync_update: bool,
```

Comment:
This shouldn't be included here, this is kind of the domain object which doesn't know about sync. You could just add a flag to `upsert()` if you need to know who is writing the document...

Reply:
You mean finalise()?

Reply:
I was just thinking of a way to set the `is_sync_update` flag because in this special case we don't want to store it with the domain data... Does this answer your question?